zone/bot.cpp

```diff
@@ -3215,14 +3215,17 @@ bool Bot::IsValidTarget(
 		return false;
 	}
 
+	SetHasLoS(DoLosChecks(tar));
```

Comment:
I'm not sure why we're adding a member variable to keep hold of LOS as LOS is prone to change at a moments notice and could become frail to other parts of the code. It's something you want to be able to calculate at that point in the code versus caching and hoping its accurate by the time you use it somewhere else.

If the goal is to reduce CPU cycles we should consider other avenues as this adds frailties to the code.